### PR TITLE
Update CONTRIBUTING.md to fix grammar errors

### PR DESCRIPTION
**Pull request by CodeMaster7000**
## PR Checklist

This PR is ready for review and meets the requirements set out
in [Suggestion how to contribute](CONTRIBUTING.md).

### DOD

- [ ] I have added new quiz{'s}
- [ ] I have added new reference link{'s}
- [x] I have made small corrections/improvements

### Changes / Instructions

This PR fixes significant grammar errors in the `CONTRIBUTING.md` file and adds more suitable titles. This is important as it makes the guide easier to follow.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 
-> After your PR is merged, you will be automatically become a contributor and will be appeared in the Readme.md as the contributor.
+> After your PR is merged, you will be automatically become a contributor and will appear in `README.md` as one.
 
-## PLEASE
+## Adding Answers
 
-> Add explanation (`or reference link`) to your answers. That will help anyone to better learn concepts they might not have a great understanding about.
+> Please add an explanation or reference link to your answers. This will help anyone to better learn concepts they 
+may not have a great understanding about.
```
Comment (Ebazhanov) [inline CONTRIBUTING.md:16]: should be on the same line (instead it will brake formatting)

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
Thanks, left one comment

File: CONTRIBUTING.md (hunk 1 above)
Comment (CodeMaster7000) [reply CONTRIBUTING.md:16]: So should I just put it on the same line?

**Comment by CodeMaster7000** [comment]
@Ebazhanov Fixed!

**Review by Ebazhanov — APPROVED** [review]
Thank you @CodeMaster7000, for your great input! 🍺

**Comment by Ebazhanov** [comment]
@all-contributors please add @CodeMaster7000 for design, code

**Pull request merged**